packages/core/src/schema/models/documentVersions.ts

```diff
@@ -21,7 +21,7 @@ export const documentVersions = latitudeSchema.table(
     documentUuid: uuid('document_uuid').notNull().defaultRandom(),
     path: varchar('path').notNull(),
     content: text('content').notNull().default(''),
-    hash: varchar('hash'),
+    resolvedContent: text('resolved_content'),
```

Comment:
`resolvedContent` is nullable. If it does have content, it is safe to use. Otherwise it's required to compute it.

In order to guarantee that this attribute is safe, it is converted to NULL each time a document from the same commit is updated / created.